Source/SwiftLintFramework/Models/Configuration.swift

```diff
@@ -269,7 +273,7 @@ private func validKeys(ruleList: RuleList) -> [String] {
     ].map({ $0.rawValue }) + ruleList.allValidIdentifiers()
 }
 
-private func warnAboutDeprecations(_ dict: [String: Any],
+private func warnAboutDeprecations(dict: [String: Any],
```

Comment:
`configuration` is probably sufficient, given that `Dictionary` is implied by the type.